packages/pharos-site/src/pages/index.tsx

```diff
@@ -36,233 +34,229 @@ const IndexPage: FC = () => {
-                </PharosLink>
-              </li>
-            </ul>
+        <div className={card}>
+          <div className={icon}>
+            <PharosIcon name="new"></PharosIcon>
           </div>
-          <div className={card}>
-            <div className={icon}>
-              <PharosIcon name="question-inverse"></PharosIcon>
-            </div>
-            <PharosHeading level="2" preset="4">
-              Support
-            </PharosHeading>
-            <p className={description}>
-              Experiencing a Pharos-related issue or just need some help? Report an issue in Github
-              or get in touch via slack.
-            </p>
-            <ul className={list}>
-              <li>
-                <PharosLink href="help" data-sc="link:help" onClick={handleLinkClick}>
-                  Help
-                </PharosLink>
-              </li>
-              <li>
-                <PharosLink href="mailto:opensource@ithaka.org">Contact us</PharosLink>
-              </li>
-              <li>
-                <PharosLink href="https://github.com/ithaka/pharos/issues" target="_blank">
-                  Report an issue
-                </PharosLink>
-              </li>
-            </ul>
+          <PharosHeading level="2" preset="4">
+            What&apos;s new: Pharos {pkg.version}
+          </PharosHeading>
+          <p className={description}>
+            Upgrade to the latest version of Pharos today to explore new possibilities for your apps
+            and projects, and help maintain consistency on JSTOR.
+          </p>
+          <ul className={list}>
+            <li>
+              <PharosLink
+                href="https://github.com/ithaka/pharos/blob/main/packages/pharos/CHANGELOG.md"
+                target="_blank"
+                data-sc="link:changelog"
+              >
+                Changelog
+              </PharosLink>
+            </li>
+            <li>
+              <PharosLink
+                href="https://pharos.jstor.org/storybooks/wc/"
+                target="_blank"
+                data-sc="link:wc storybook"
+              >
+                Web Components Storybook
+              </PharosLink>
+            </li>
+            <li>
+              <PharosLink
+                href="https://pharos.jstor.org/storybooks/react/"
+                target="_blank"
+                data-sc="link:react storybook"
+              >
+                React Components Storybook
+              </PharosLink>
+            </li>
+          </ul>
+        </div>
+        <div className={card}>
+          <div className={icon}>
+            <PharosIcon name="question-inverse"></PharosIcon>
           </div>
+          <PharosHeading level="2" preset="4">
+            Support
+          </PharosHeading>
+          <p className={description}>
+            Experiencing a Pharos-related issue or just need some help? Report an issue in Github or
+            get in touch via slack.
```

Comment:
Or should this be removed altogether?

```suggestion
            get in touch via Slack.
```